### PR TITLE
SmartDetector package documentation

### PR DESCRIPTION
**Pull request by efvink**
fixing a bug - SmartDetector SDK documentation isn't visible to the user when using SmartDetector package

### CODE REVIEW

File: src/SDK/SmartDetectorsShared/Clients/TelemetryDataClientCreationException.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
-namespace Microsoft.Azure.Monitoring.SmartDetectors.Clients
```
Comment (EranGon) [inline src/SDK/SmartDetectorsShared/Clients/TelemetryDataClientCreationException.cs]: >osoft.Azure.Monitoring.SmartDetectors.Clients [](start = 14, length = 45)

why did you move it?

File: src/SDK/SmartDetectorsSDK/SmartDetectorsSDK.csproj [hunk 2]
```diff
@@ -6,6 +6,12 @@
 	  <StyleCopTreatErrorsAsWarnings>False</StyleCopTreatErrorsAsWarnings>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>bin\Debug\net461\Microsoft.Azure.Monitoring.SmartDetectors.xml</DocumentationFile>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>bin\Release\net461\Microsoft.Azure.Monitoring.SmartDetectors.xml</DocumentationFile>
```
Comment (EranGon) [inline src/SDK/SmartDetectorsSDK/SmartDetectorsSDK.csproj]: >bin\Release\net461 [](start = 23, length = 18)

can't you use $(OutDir)?

File: src/SDK/SmartDetectorsShared/Clients/TelemetryDataClientCreationException.cs (hunk 1 above)
Comment (efvink) [reply src/SDK/SmartDetectorsShared/Clients/TelemetryDataClientCreationException.cs]: TelemetryDataClientCreationException used in IAnalysisServicesFactory

---
In reply to: [175494337](https://github.com/liadtal/Smart-Alerts/pull/111#discussion_r175494337) [](ancestors = 175494337)
